resources/netregexes.ts

```diff
@@ -1,11 +1,13 @@
-import { NetRegex } from 'types/trigger';
+import { BaseRegExp } from '../types/trigger';
```

Comment:
Should this still be 'types/trigger' and not the relative path?

Reply:
I am not sure how that was working, so I'm going to stick with the relative path import.